content/_data/books/jenkins2-up-and-running.yml

```diff
@@ -0,0 +1,19 @@
+title: "Jenkins 2: Up and Running"
+url: https://www.amazon.com/Jenkins-Deployment-Pipeline-Generation-Automation/dp/1491979593/
+author: "Brent Laster"
+publication_year: 2018
+publisher: "O'Reilly Media"
+type: Paid
+image: books/jenkins2-up-and-running.jpeg
+description: |-
+  Design, implement, and execute continuous delivery pipelines with a level of flexibility, control, and ease of maintenance that was not possible with Jenkins before. With this practical book, build administrators, developers, testers, and other professionals will learn how the features in Jenkins 2 let you define pipelines as code, leverage integration with other key technologies, and create automated, reliable pipelines to simplify and accelerate your DevOps environments.
+  Author Brent Laster shows you how Jenkins 2 is significantly different from the more traditional, web-only versions of this popular open source automation platform. If you’re familiar with Jenkins and want to take advantage of the new technologies to transform your legacy pipelines or build new modern, automated continuous delivery environments, this is your book.
+  1. Create continuous delivery pipelines as code with the Jenkins domain-specific language
+  2. Get practical guidance on how to migrate existing jobs and pipelines
+  3. Harness best practices and new methods for controlling access and security
+  4. Explore the structure, implementation, and use of shared pipeline libraries
+  5. Learn the differences between declarative syntax and scripted syntax
+  6. Leverage new and existing project types in Jenkins
+  7. Understand and use the new Blue Ocean graphical interface
+  8. Take advantage of the capabilities of the underlying OS in your pipeline
+  9. Integrate analysis tools, artifact management, and containers
```

Comment:
```suggestion
  Author Brent Laster shows you how Jenkins 2 is significantly different from the more traditional, web-only versions of this popular open source automation platform. If you’re familiar with Jenkins and want to take advantage of the new technologies to transform your legacy pipelines or build new modern, automated continuous delivery environments, this is your book.
  
  . Create continuous delivery pipelines as code with the Jenkins domain-specific language
  . Get practical guidance on how to migrate existing jobs and pipelines
  . Harness best practices and new methods for controlling access and security
  . Explore the structure, implementation, and use of shared pipeline libraries
  . Learn the differences between declarative syntax and scripted syntax
  . Leverage new and existing project types in Jenkins
  . Understand and use the new Blue Ocean graphical interface
  . Take advantage of the capabilities of the underlying OS in your pipeline
  . Integrate analysis tools, artifact management, and containers
```
just adding a space between the description and numbered steps
In ascii, the lists will number themselves if just a period is used. The numbers do not harm anything, but are redundant in this case.